UnityProject/Assets/Scripts/Systems/InGameEvents/InGameEventScripts/EventScrubberSurge.cs

```diff
@@ -8,12 +8,19 @@
 using Strings;
 using ScriptableObjects;
 using Objects.Atmospherics;
+using UnityEngine.Serialization;
 
 
 namespace InGameEvents
 {
 	public class EventScrubberSurge : EventScriptBase
 	{
+		//TODO some time
+		//5% chance to make the janitor antagonist When the scrubber Surge  event happens
+		//like a special antag "you can't deal with this crap anymore! make a mess!!!"
+
+		private System.Random RNG = new System.Random();
```

Comment:
private readonly System.Random RNG = new System.Random();

Is the only codacy complaint, may aswell.
Also codacy didn't flag this just a pet peeve for me. 'random' might be a better name than 'RNG'.

Reply:
RNG is used in multiple places